spec_7/index.js

```diff
@@ -0,0 +1,36 @@
+import { identifiers } from './resources'
+import converter from './dateConverter.js'
+
+const dateFile = 'spec_7/dates.txt'
+const dates = converter.getRawDates( dateFile )
+const validFormats = converter.getValidFormats( dates )
+
+const invalidFormats = {
+  slashies: converter.filterFormats( dates, identifiers.SLASH ),
+  hashies: converter.filterFormats( dates, identifiers.HASH ),
+  asteries: converter.filterFormats( dates, identifiers.ASTERISK ),
+  spacies: converter.filterFormats( dates, identifiers.SPACEY )
+}
+
+const convertTheInvalid = theInvalids => {
```

Comment:
`theInvalids` is a tricky parameter name - sine you replace values in this set with other values, they're not always invalid are they?  That, or I totally misunderstand the intent of the code below...